docs/swagger.yaml

```diff
@@ -409,6 +397,27 @@ components:
             barcode:
               format: EAN-13
               data: '123456'
+      properties:
+        orderLines:
+          type: array
+          description: 'All numbers are interpreted as integers with two trailing zeros. ex: 10.00 = 1000. Amounts are specified in minor units. For Norwegian kroner (NOK) that means 1 kr = 100 øre. Example: 10 kr = 1000 øre.'
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/OrderLine'
+        bottomLine:
+          $ref: '#/components/schemas/BottomLine'
+        merchantInfo:
+          type: object
+          properties:
+            isRegisteredInMvaForetakregisteret:
+              type: boolean
+              description: Is registered in the register of business enterprises
+              nullable: true
+          required:
+            - isRegisteredInMvaForetakregisteret
```

Comment:
this can be required for now, but it should be optional as we might need different booleans for each country.